locale/te.yml

```diff
@@ -1253,318 +1253,629 @@ __TRAIT__: ""
     # ✅ @KarthikMothiki
+    # 👀 @sujithkumardola
   nonlocal: ""
     # 👀 @nagendrady
     # 👀 @manojkrishnak
     # 🤔 @Ln11211 A translation is difficult for this, but "నాన్ లోకల్" seems ok.
     # 🤔 @KarthikMothiki స్థానికేతర is the exact word for non-local but many people would prefer hard-translation itself.
+    # 👀 @sujithkumardola
   not: "లేదు"
     # ✅ @nagendrady
     # ✅ @manojkrishnak
     # ✅ @Ln11211
     # ✅ @KarthikMothiki
+    # 👀 @sujithkumardola
   or: "లేదా"
     # ✅ @nagendrady
     # ✅ @manojkrishnak
     # ✅ @Ln11211
     # ✅ @KarthikMothiki 
+    # 👀 @sujithkumardola
   pass: "పాస్"
     # ✅ @nagendrady
     # ❓ @manojkrishnak Here true translation is written, there is an better alternative, that's అందించుట.
     # ✅ @Ln11211
     # ✅ @KarthikMothiki
+    # 👀 @sujithkumardola
   raise: "పెంచండి"
     # ✅ @nagendrady
     # ✅ @manojkrishnak
     # ✅ @Ln11211
     # ❓ @KarthikMothiki when it comes to hard translation of the context ఎత్తుట will work good
+    # 👀 @sujithkumardola
   return: "తిరిగి వచ్చు"
     # ✅ @nagendrady
     # ❓ @manojkrishnak జవాబు ఇచ్చు, బయటకి వచ్చుట may be considered as better translations depending upon the usage(return some value or come out of the loop/if condition).
     # ❓ @Ln11211 "తిరిగిఇచ్చుట" seems apt for the "return" keyword
     # ✅ @KarthikMothiki
+    # 👀 @sujithkumardola
   try: "ప్రయత్నించండి"
     # ✅ @nagendrady
     # ✅ @manojkrishnak
     # ❓ @Ln11211 "ప్రత్నించు" is better translation for "try"
     # ✅ @KarthikMothiki
+    # 👀 @sujithkumardola
   while: "అయితే"
     # ✅ @nagendrady
     # ✅ @manojkrishnak
     # ✅ @Ln11211
     # ✅ @KarthikMothiki
+    # 👀 @sujithkumardola
   with: "తో"
     # ✅ @nagendrady
     # ✅ @manojkrishnak
     # ✅ @Ln11211
     # ✅ @KarthikMothiki
+    # 👀 @sujithkumardola
   yield: "లభించు"
     # ✅ @nagendrady
     # ❓ @manojkrishnak May be a better translation is required.
     # ❓ @Ln11211 Would recommend using "లబించుట", "లభించు" is also fine.
     # ✅ @KarthikMothiki
+    # 👀 @sujithkumardola
 
 # << Built-In Functions >>
 
   # @KarthikMothiki: Some of the keywords are hard-translated!
   abs: "సంపూర్ణ సంఖ్య"
-  # ✅ @KarthikMothiki
+    # 👀 @nagendrady
+    # 👀 @manojkrishnak
+    # 👀 @Ln11211
+    # ✅ @KarthikMothiki
+    # ✅ @sujithkumardola
   all: "అన్ని"
-  # ✅ @KarthikMothiki
+    # 👀 @nagendrady
+    # 👀 @manojkrishnak
+    # 👀 @Ln11211
+    # ✅ @KarthikMothiki
+    # ✅ @sujithkumardola
   any: "ఏదైనా"
-  # ✅ @KarthikMothiki
+    # 👀 @nagendrady
+    # 👀 @manojkrishnak
+    # 👀 @Ln11211
+    # ✅ @KarthikMothiki
+    # ✅ @sujithkumardola
   ascii: "ఆశ్కి"
-  # 👀 @KarthikMothiki there is no word for ascii in telugu
+    # 👀 @nagendrady
+    # 👀 @manojkrishnak
+    # 👀 @Ln11211
+    # 👀 @KarthikMothiki there is no word for ascii in telugu
+    # ✅ @sujithkumardola "ఆశ్కి" looks fine
   bin: "బైనరీ"
-  # 👀 @KarthikMothiki
+    # 👀 @nagendrady
+    # 👀 @manojkrishnak
+    # 👀 @Ln11211
+    # 👀 @KarthikMothiki
+    # 👀 @sujithkumardola
```

Comment:
I typically use the 👀 emoji to denote when someone has left a translation blank. If you're having a hard time with a translation, I suggest using the 🤔 emoji instead.